saleor/graphql/discount/tests/queries/test_sales.py

```diff
@@ -108,6 +122,7 @@ def test_sales_query(
     channel_USD,
     permission_manage_products,
 ):
+    SaleToPromotionConverter.convert_sales_to_promotions()
```

Comment:
I don't think we should use `SaleToPromotionConverter` to convert any data in tests. Currently, tests create sales and save them to the database. In the next steps, we convert nearly create sales to promotions. We should fix it in all tests.